.github/workflows/npm-publish.yml

```diff
@@ -14,6 +14,7 @@ jobs:
           node-version: '16.x'
           registry-url: 'https://registry.npmjs.org'
       - run: npm install
-      - run: npm publish --workspaces
+      - run: npm run build
```

Comment:
nit: I saw we had `release` in `package.json`. do we want to run it rather than `build` here?